src/naming.jl

```diff
@@ -173,6 +177,13 @@ access(c, keys...) = access(access(c, keys[1]), Base.tail(keys)...)
 access(c::AbstractDict, key) = getindex(c, key)
 access(c, key) = getproperty(c, key)
 
+"""
+    allignore(c)
+Return all the keys `c` will be ignored during [`savename`](@ref).
+This is an empty tuple by default.
+"""
+allignore(c::Any) = ()
```

Comment:
I would rename this to `nonignore`. 
`allaccess` mean access all field, then `allignore` means ignore all fields.

Reply:
And you don't need to specify `::Any`, it's the default.

Reply:
Hm, I am in favor of `allignore`. `allaccess` does not mean "access all the fields", but it means "all the fields that will be accessed". It just so happens that by default it _is_ all fields.

Similarly `allignore` means "all the fields that will be ignored". `noignore` isn't so intuitive, because I would naively think that it would return that fields that _wouldn't_ be ignored...

Reply:
Ok. I mean it's a function thats not exported anyway so I'm fine with `allignore`

Reply:
I thought the variabels have the same meaning as @Datseris explains. I will keep it as `allignore`.